_posts/2017-05-26-deploying-create-react-app-to-s3.markdown

```diff
@@ -0,0 +1,267 @@
+---
+layout: post
+title:  "Deploying create-react-app via S3"
+date:   2017-06-01 00:00:00 -0000
+categories: react devops
+author: Zach Cotter
+---
+
+At Forward Financing, we've started using
+[create-react-app][create-react-app]
+as a starter kit for all of our new React projects. Create-React-App is awesome
+because it comes with a one size fits all webpack configuration via the
+`react-scripts` package. All you have to do is `npm run build` and you have a
+ready to ship `/build` folder with minified production asset files.
+
+The first project we did this with was a front-end for an application used internally to manage our user's roles and access to pages across our suite
```

Comment:
Go with active voice here, ie: 

For our first project using create-react-app, we built an interface to manage our users' roles and access across our tech ecosystem. (one would assume managing users roles and access is an internal process)